src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -781,24 +778,14 @@ public WorldEnvelope getEnvelope() {
    */
   public void calculateTransitCenter() {
     if (hasTransit) {
-      TDoubleList latitudes = new TDoubleLinkedList();
-      TDoubleList longitudes = new TDoubleLinkedList();
-      Median median = new Median();
+      Envelope tempEnvelope = new Envelope();
 
       getVerticesOfType(TransitStopVertex.class)
-        .stream()
         .forEach(v -> {
-          latitudes.add(v.getLat());
-          longitudes.add(v.getLon());
+          tempEnvelope.expandToInclude(v.getLon(), v.getLat());
         });
 
-      median.setData(latitudes.toArray());
-      double medianLatitude = median.evaluate();
-      median = new Median();
-      median.setData(longitudes.toArray());
-      double medianLongitude = median.evaluate();
-
-      this.center = new Coordinate(medianLongitude, medianLatitude);
+      this.center = tempEnvelope.centre();
```

Comment:
This changes the behaviour, as previously the median coordinates were reported, while this returns the center of the envelope. The difference can be quite big when you have even one outlying stop.

Reply:
hmm, median I taught it refers to mid of segment defined by latitude/longitude points, but will dig into it more to make sure no functional changes are happening

Reply:
The center of the envelope is the mean, not the median.